pokemongo_bot/cell_workers/sniper.py

```diff
@@ -277,8 +278,11 @@ def is_snipeable(self, pokemon):
                 if pokemon.get('vip', False):
                     self._trace('{} is not catchable and bad IV (if any), however its a VIP!'.format(pokemon.get('pokemon_name')))
                 else:
-                    self._trace('{} is not catachable, nor a VIP and bad IV (if any). Skipping...'.format(pokemon.get('pokemon_name')))
-                    return False
+                    if pokemon['missing']:
```

Comment:
Please change it from `if pokemon['missing']:` to `if pokemon.get('missing', False):`.